### ISSUE TITLE
Safari camera settings not working correctly with rc.2

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/ratio-and-proportion#543]
**Test device**
MacBook Air (m1 chip)

**Operating System**
macOS 13.0.1

**Browser**
safari 16.1

**Problem description**
For https://github.com/phetsims/qa/issues/863, I have my safari camera setting set to "ask," but when I add ?cameraInput=hands, nothing happens. I have to go into my safari settings and change to "allow" in order for the camera to turn on. I didn't have this issue with rc.1. The camera works as it should.

**Visuals**

https://user-images.githubusercontent.com/87318828/207429133-3ecd0c7f-1424-4d84-9295-642c5b0780df.mp4

<img width="797" alt="Screenshot 2022-12-13 at 2 44 06 PM" src="https://user-images.githubusercontent.com/87318828/207429422-cacc3a86-9f97-4396-9c99-bd7fc5f2c10a.png">

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Ratio and Proportion‬
URL: https://phet-dev.colorado.edu/html/ratio-and-proportion/1.2.0-rc.2/phet/ratio-and-proportion_all_phet.html
Version: 1.2.0-rc.2 2022-12-05 22:59:52 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36
Language: en-US
Window: 1581x816
Pixel Ratio: 1.7999999523162842/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 31 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}
</details>

**Comment by Nancy-Salpepi** [comment]
I am seeing the same issue with macOS 10.15.7 + safari 15.6.1. With my safari camera settings set to "ask" for the phet-dev website, I get this message with rc.1, but not rc.2:
<img width="312" alt="Screenshot 2022-12-13 at 2 23 22 PM" src="https://user-images.githubusercontent.com/87318828/207707654-0b4f0d76-4ed2-44c8-856e-1c51d554984e.png">

**Comment by zepumph** [comment]
I was not able to make much progress on this today, next step is to test if this works on my iPad.

**Comment by zepumph** [comment]
@jessegreenberg was able to help me debug this and found that in many cases, the deviceID from the browser is `''`, so the default value of most device ids (which is also the empty string) often doesn't trigger it to start the stream.

@jessegreenberg and I couldn't get to the bottom of why this bug was occurring in rc.2 but not rc.1, but we have confirmed this fixed on master.

Also note that this was effecting all devices we tested, including chrome and windows. The reason it wasn't showing up often was because most of our testing links had already "allowed" camera, so it didn't automatically reproduce until resetting the camera permission. Ready for cherry pick.
